tests/phpunit/tests/media.php

```diff
@@ -3973,6 +3973,54 @@ public function test_wp_generate_attachment_metadata_doesnt_generate_sizes_for_1
 		);
 	}
 
+	/**
+	 * @ticket 58212
+	 * @covers ::wp_get_attachment_image()
+	 */
+	public function test_wp_get_attachment_image_context_filter_default() {
+		$last_context = '';
+		add_filter(
+			'wp_get_attachment_image_context',
+			static function( $context ) use ( &$last_context ) {
+				$last_context = $context;
+				return $context;
```

Comment:
Instead of using a closure here, why not use use a method and set a class property?

Reply:
Similar question as above, why would that be beneficial? Adding a class property adds things to the class when it's only used by very few specific tests, so that seems not worthwhile. Other WP core test code has often been doing that just because it was written when we were still supporting PHP 5.2.

Reply:
Okay 👍